### ISSUE TITLE
Pointable Bug

### ISSUE THREAD

**Issue by surgiie** [surgiie/compean#19]
The pointable package is using lists on Transaction.php which is no longer supported on L5.3 Query Builder, so just change it to pluck for now. I have submitted a pull request to the package owner.

**Comment by yarwest** [comment]
So this is where those problems I had came from...... glad you solved it

**Comment by surgiie** [comment]
Yeah youll have to change it since its in the vendor folder but i did submit a pull request so the package could be updated as well
